prow/manifests/overlays/metal3/config.yaml

```diff
@@ -2735,6 +2735,11 @@ presubmits:
     agent: jenkins
     always_run: false
     optional: true
+  # {job_prefix}-ironic-image-build-test
+  - name: metal3-ironic-image-build-test
+    agent: jenkins
+    always_run: false
```

Comment:
How much time does the build take, do we want to make it always run. or just make it required?

Reply:
I think it takes ~15 min  I am not sure if it will be practical to run automatically every push

Reply:
BMO e2e runs for 20 and/or 40 mins, and those get run every time something changes, and Ironic-image has less traffic ? @kashifest thoughts?